api/src/main/java/org/openmrs/module/attachments/AttachmentsConstants.java

```diff
@@ -69,6 +73,9 @@
 	
 	public static final String UNKNOWN_MIME_TYPE = "application/octet-stream";
 	
+	public static final List<String> EXECUTABLE_MIME_TYPES = Arrays.asList("application/vnd.microsoft.portable-executable",
```

Comment:
i have decided to use exact mime type like for .exe file and js files because;
- application/octet-stream is too specific 
- application/octet-stream has been to test very many cases.  I decided to drop it.

Reply:
That's most likely not sufficient, it's Windows-minded, and many installations out there are Un*x-based.
Just Googling a little bit gives me those guys for instance:
* `application/sql` (.sql)
* `application/x-sh` (.sh)
* `application/java-archive` (.jar)

Think of VB/VBA scripts, Python scripts (see [this](https://stackoverflow.com/a/45614613/321797) for instance), ... etc etc.